src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -59,6 +67,62 @@ internal void InitializeLicense(string licenseText)
             license = foundLicense;
         }
 
+        static bool FirstTimeUser()
+        {
+            // Check for the presence of HKCU:Software/NServiceBus
+            using (var regNsbRoot = Registry.CurrentUser.OpenSubKey(@"Software\NServiceBus"))
+            {
+                if (regNsbRoot == null)
+                {
+                    //Check for the presence of HKCU:SOFTWARE\ParticularSoftware
+                    using (var regParticularRoot = Registry.CurrentUser.OpenSubKey(@"Software\ParticularSoftware"))
+                    {
+                        if (regParticularRoot == null)
+                        {
+                            return true;
+                        }
+                    }
+                }
+            }
+            return false;
+        }
+
+        /// <summary>
+        /// Adding the async void to explicitly have a fire and forget model for invoking the web api.
+        /// </summary>
+        async void TrackFirstTimeUsageEvent()
```

Comment:
> Async void is for fire & forget asynchronous IO bound operations. 

the "approved" use case for async void is mainly to enable async operations in callbacks. When calling this method, there is no real sign stating the fire & forget nature of this operation which may lead a caller to believe the operation completed successfully after returning from the call. Task.Run makes the "fire and forget" nature more obvious. I'm also good with your `Ignore` suggestion.